patches/react-native-mmkv-storage+0.6.6.patch

```diff
@@ -28,3 +28,16 @@ index ffac535..4608563 100644
          return Value::undefined();
      });
      
+diff --git a/node_modules/react-native-mmkv-storage/android/build.gradle b/node_modules/react-native-mmkv-storage/android/build.gradle
+index 618a7de..4aaca89 100644
+--- a/node_modules/react-native-mmkv-storage/android/build.gradle
++++ b/node_modules/react-native-mmkv-storage/android/build.gradle
+@@ -85,7 +85,7 @@ android {
+   }
+ 
+   packagingOptions {
+-    excludes = ["**/libc++_shared.so","**/libjsi.so","**/libreactnativejni.so"]
++    excludes = ["**/libc++_shared.so","**/libjsi.so","**/libreactnativejni.so", "META-INF/MANIFEST.MF"]
```

Comment:
Fix for the error encountered at https://github.com/mattermost/mattermost-mobile/pull/5879#issuecomment-995688248.
Already on package's master but not yet released - https://github.com/ammarahm-ed/react-native-mmkv-storage/commit/5dce2e9b6930cec1aa73843a78dedb4fa0545819